docker-compose.yml

```
@@ -25,6 +25,10 @@ services:
     logging: *default-logging
     environment:
       - SIA_MODULES=gctwra
+      - SKYD_DISK_CACHE_DISABLED=true
```

```suggestion
      - SKYD_DISK_CACHE_ENABLED=false
```

Can we use enabled variable and set it to false if it's not enabled instead? The inversed env variable pattern, similar to what we do with JAEGER seems unintuitive. Also, is the disk cache enabled by default?